tools/wptserve/wptserve/handlers.py

```diff
@@ -300,11 +300,23 @@ def _set_path_and_load_file(self, request, response, func):
         """
         path = filesystem_path(self.base_path, request, self.url_base)
 
-        sys_path = sys.path[:]
-        sys_modules = sys.modules.copy()
+        # Add root of WPT to the path, for py file handler imports.
+        # TODO: Find a better location to do this. For some reason for the
+        # tests I had to do it here... even the __init__ didn't work.
+        # TODO: When running 'wpt run', this is already on the path. But it
+        # isn't for tests, so need to figure out the correct story here.
+        # TODO: Maybe want to add the docroot rather than _WPT_ROOT?
+        _WPT_ROOT = os.path.abspath(os.path.join(
```

Comment:
Yeah, we shouldn't assume that wptserve is only used for wpt; that isn't quite true. So we don't want to hardcode the path here.